scripts/generateTS.sh

```diff
@@ -230,9 +247,50 @@ export {
   CanvasCommentLink
 } from \"./canvas-info-v3.ts\";
 export {
-  HttpsApiDataplatformIbmComSchemasCommonCanvasPalettePaletteV3SchemaJson as PipelineFlowPalette,
+  PipelineFlowPalette,
   CategoryDef
-} from \"./palette-v3.ts\";"
+} from \"./palette-v3.ts\";
+export {
+  ParameterDefinitions,
+  ConditionsDefinition,
+  MessageDefinition,
+  EvaluateDefinition,
+  ValidationDefinition,
+  FailMessageDefinition,
+  AndDefinition,
+  OrDefinition,
+  ConditionDefinition,
+  FilterConditionDefinition,
+  EnabledDefinition,
+  VisibleDefinition,
+  FilterDefinition,
+  EnumFilterDefinition,
+  AllowChangeDefinition,
+  DefaultValueDefinition,
+  ParameterDefinition as OperatorParameterDefinition,
+  ParameterRefDefinition,
+  ComplexTypeDefinition as OperatorComplexTypeDefinition,
+  UIHints,
+  GroupDefinition,
+  ParameterDefinition1 as ParameterDefinition,
+  ComplexTypeDefinition1 as ComplexTypeDefinition,
```

Comment:
Is it possible to rename these objects in the schemas so we don't have to do the 'rename' here when it is exported?
```
  ParameterDefinition as OperatorParameterDefinition,
  ComplexTypeDefinition as OperatorComplexTypeDefinition,
  ParameterDefinition1 as ParameterDefinition,
  ComplexTypeDefinition1 as ComplexTypeDefinition,
  ```

Reply:
@matthoward366 Is this going to be a breaking change?

- Changing [this line](https://github.com/elyra-ai/pipeline-schemas/blob/main/common-pipeline/operators/operator-v3-schema.json#L68) to `"operator_parameter_definition"`
- Changing [this line](https://github.com/elyra-ai/pipeline-schemas/blob/main/common-pipeline/operators/operator-v3-schema.json#L163) to `"operator_complex_type_definition"`

Reply:
I will make this change. I didn't find any "parameter_definition" and "complex_type_definition" in [existing paramDefs](https://github.com/elyra-ai/canvas/tree/main/canvas_modules/harness/test_resources/parameterDefs).